packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -1236,6 +1250,104 @@ class RenderEditable extends RenderBox {
     handleLongPress();
   }
 
+  TextPosition _lastDragSelectionStartPosition;
+
+  /// If [ignorePointer] is false (the default) then this method is called by
```

Comment:
The first sentence in a doc comment should be a summary of what the function does. This describes the conditions under which it is called (which is good to document, but it's not a summary of what it does).